src/main/java/com/ericsson/ei/queryservice/ProcessFailedNotification.java

```diff
@@ -68,24 +72,25 @@
             }
             return null;
         }).collect(Collectors.toList());
-
     }
 
     /**
-     * The method is responsible for the delete the missed notification using subscription name
+     * Delete a failed notification object by using the subscriptionName key.
      *
      * @param subscriptionName
      * @return boolean
      */
-    public boolean deleteMissedNotification(String subscriptionName) {
+    public boolean deleteFailedNotification(String subscriptionName) {
```

Comment:
delete failed notification is not used any more and may be removed.

Reply:
Done